### DISCUSSION TITLE
build: How to build with gcov support?

### DISCUSSION THREAD

**Discussion by osevan** [netblue30/firejail#5059] [Q&A] [answered]
How i can pass flag to configure script for gcov support?

Second question :

When I build with gcov support, firejail build routine collecting gcda profiles amd building automaticaly again with profile? 

Or I need pass second build stage with different configure flag? 

Thanks and

Best regards

**Comment by rusty-snake** [comment, marked as answer]
```console
$ ./configure --help
...
  --enable-gcov           Gcov instrumentation
...
```

**Reply by osevan** [reply]
> ```
> $ ./configure --help
> ...
>   --enable-gcov           Gcov instrumentation
> ...
> ```

Second compile with instrumentated profiles including this flag ?

**Comment by osevan** [comment]
Or first build with gcov and install, after then execute gcov.sh and install build again?

**Comment by osevan** [comment]
Or should I add on stage 2 with -fprofile-use.?

**Comment by osevan** [comment]
i thougt gcov is needed for pgo builds...
